EssentialApp/EssentialApp/FeedUIComposer.swift

```diff
@@ -12,7 +12,8 @@ public final class FeedUIComposer {
 	
 	public static func feedComposedWith(
 		feedLoader: @escaping () -> FeedLoader.Publisher,
-		imageLoader: @escaping (URL) -> FeedImageDataLoader.Publisher
+		imageLoader: @escaping (URL) -> FeedImageDataLoader.Publisher,
+				onImageSelection: @escaping (FeedImage)->()
```

Comment:
Fix indentation and follow the codebase convention with the closure type:

```suggestion
		imageLoader: @escaping (URL) -> FeedImageDataLoader.Publisher,
		onImageSelection: @escaping (FeedImage) -> Void
```